src/main/java/com/nylas/NylasClient.java

```diff
@@ -224,12 +261,19 @@ public NylasClient build() {
 			return new NylasClient(httpClient, baseUrl);
 		}
 	}
+
+	static enum AuthMethod {
+		BASIC,
+		BEARER,
+		;
+	}
 	
 	static enum HttpMethod {
 		GET,
 		PUT,
 		POST,
 		DELETE,
+		PATCH,
 		;
```

Comment:
nit: Do you need the `;` at the end here...? It looks weird from a formatting perspective.

Reply:
Thank you! Done ✅